vlib/v/gen/native/tests/assert.vv

```diff
@@ -0,0 +1,10 @@
+fn main() {
+	assert 1 == 1
+	assert 3 == 3
+	assert 1 == 1
+	assert 3 == 3
+	if 1 == 2 {
+		assert 1 == 2
```

Comment:
hm, I get `assert pass` even for `./v -b native run x.v`:
```v
fn main() {
    assert 1 == 2
    println('assert pass')
}
```

Reply:
`./v run x.v` produces:
```
0[20:33:26]delian@nemesis: (native-assert) /v/vmaster $ ./v run x.v
x.v:2: FAIL: fn main.main: assert 1 == 2
   left value: 1 = 1
  right value: 2
V panic: Assertion failed...
v hash: b5de5d2
/tmp/v/x.1052528661154133212.tmp.c:5708: at _v_panic: Backtrace
/tmp/v/x.1052528661154133212.tmp.c:10222: by main__main
/tmp/v/x.1052528661154133212.tmp.c:10580: by main
1[20:33:27]delian@nemesis: (native-assert) /v/vmaster $
```

Reply:
fixed